lib/forwardable.rb

```diff
@@ -154,9 +150,7 @@ def instance_delegate(hash)
   def def_instance_delegators(accessor, *methods)
     methods.delete("__send__")
     methods.delete("__id__")
-    for method in methods
-      def_instance_delegator(accessor, method)
-    end
+    methods.each { |method|  def_instance_delegator(accessor, method) }
```

Comment:
I'm probably one of the very few people who don't make changes like this, but I don't think there's anything wrong in `for ... in ...`. This works perfectly fine, and when both of them are equally great you don't have to stick to one of them. Commodity doesn't seem like a huge issue here unless it is as uncommon as [the flip flop operator](https://bugs.ruby-lang.org/issues/5400) is. I would also like to bring up [this comment in Rails about styling changes](https://github.com/rails/rails/pull/13771#issuecomment-32746700). I don't really mean to reject you, but I wanted to be 100% transparent about why I raised the points I've raised.

Reply:
Thank you for so frank answer, it is just my first contribution in open source, I didn't think about things you have raised, but now I will.
I know, that syntax changes doesn't make big improvements, but when I have creating this PR I have next motivation:
I have reading POODR book by Sandi Metz, and one chapter explains how to create composition via Forwardable module, I thought, that syntax like:
```ruby
class A
  extend Forwardable
  # I think, it's not very good write so :@object
  def_delegators :@object, :do_somethhing 
  # Better using access methods like :object, via attr_accessor(or reader)
  def_delegators :object, :do_something
end
```
I started to learn source code to know, can I write so,  and seen for loop, it was committed at 2011, it really so old commit, I thought, that I can help a little bit improve this gem, because I think better using ruby standard libs than thousands one-day gems with bad support for solution any problems.

Reply:
Thanks @ioquatix for jumping in, and I'm sorry @woarewe if I was too harsh. I really didn't mean to ket you down. Now that you updated the code I see that there is value in this PR. Your contribution is much appreciated and the community will surely benefit from it ❤️